GameData/RationalResources/CRP/Atm_GasIV.cfg

```diff
@@ -88,7 +88,7 @@ PLANETARY_RESOURCE:NEEDS[CommunityResourcePack,!ClassicStock]
 		Dispersal = 0
 	}
 }
-PLANETARY_RESOURCE
+PLANETARY_RESOURCE:NEEDS[ClassicStock]
```

Comment:
Actually, I may rather these (all instances of MetalOre placement in this CRP folder) be deleted, or left untouched as legacy. This reminds me, the Classic Stock templates also have placement configs for MetalOre. I should do something about those.